pandas/__init__.py

```diff
@@ -57,3 +56,12 @@
 from pandas.util.print_versions import show_versions
 import pandas.util.testing
 
+# use the closest tagged version if possible
+from ._version import get_versions
+v = get_versions()
+__version__ = v.get('closest-tag',v['version'])
+del get_versions, v
+
+from ._version import get_versions
```

Comment:
What is going on here? This looks like a variation of the above lines... I assume you meant to remove one or the other?